helm/ecowitt2mqtt/values.yaml

```diff
@@ -0,0 +1,147 @@
+# Default values for ecowitt2mqtt.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: bachya/ecowitt2mqtt
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+mqtt:
+  auth:
+    enabled: false
+    username: username
+    password: password
+
+envvar:
+  #- name: ECOWITT2MQTT_BATTERY_OVERRIDE
```

Comment:
Why are these variables commented out?

Reply:
The default values i was setting didnt work and I wasnt using them, I left if for someone who needs them to uncomment and set as required, I could remove them completely as they are builtin but i put them there so people would see where to put them rather than have to figure it out from the docs. 

```
- name: ECOWITT2MQTT_BATTERY_OVERRIDE
  value: numeric
```

results in `2022-09-02 07:30:01,686 | ecowitt2mqtt | ERROR | Unable to parse battery configurations: numeric`

Again I wanted to get the PR up for comments sooner but will look into the code to see how you are parsing this to see why it is erroring.